test/e2e/tests/ens.spec.js

```diff
@@ -25,6 +25,34 @@ describe('ENS', function () {
         };
       });
 
+    await mockServer
+      .forPost(infuraUrl)
+      .withJsonBodyIncluding({ method: 'eth_getBalance' })
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: {
+            jsonrpc: '2.0',
+            id: '1111111111111111',
+            result: '0x1',
+          },
+        };
+      });
+
+    await mockServer
+      .forPost(infuraUrl)
+      .withJsonBodyIncluding({ method: 'eth_getBlockByNumber' })
```

Comment:
Any idea why these changes were needed here? I see that we've changed the timing of some of the network calls in the network controller, so maybe that destabilized this test if it was already flaky, but I don't see a clear connection.

Reply:
Yes. So this request should have been mocked all along, but previously that was not a problem. The reason why this test was failing was because the loading spinner that appears when you load MetaMask never disappeared. Previously, it would disappear once the `network` state changed from "loading" to the network ID, which happened when `net_version` returned with a successful response. A request to `eth_getBlockByNumber` was also made in order to get the latest block to determine EIP-1559 support, but that request was not awaited, and it didn't impact whether `network` was updated or not. Now, requests to `net_version` and `eth_getBlockByNumber` are made simultaneously, and updating `networkStatus` (which the spinner now uses to determine the loaded status) relies on both requests succeeding. Hence why I now have to mock this request.

Reply:
Ah, yes, that's true. That's to prevent noise from showing up when running the test.